Eclipse Perform Setup Tasks fails with Cmder

### Expected behavior

As a user of Eclipse, I want to perform the Setup Tasks without errors.


### Actual behavior

Eclipse throws an java.io.FileNotFoundException: C:\Eclipse\temp\cmder.zip (The system cannot find the path specified).

![grafik](https://user-images.githubusercontent.com/28297839/45619862-1ae8ac80-ba7b-11e8-9047-45f79f8f6351.png)



### Steps to reproduce (bug) / Use Case of feature request (enhancement)

Run the Setup Tasks by clicking the Button in the Toolbar and select the Checkbox "unzip unzip.cmder"

This Error occured in my and @jdiazgon s Eclipse.


### Workaround

Unselect the Checkbox "unzip unzip.cmder".

```java
Executing manual tasks
Java(TM) SE Runtime Environment 1.8.0_121-b13
Product devonfw.cobigen.development.ide.Neon
Workspace C:\CobiGenIDE\workspaces\cobigen-development
Project devonfw.tools.cobigen.develop
Bundle com.github.maybeec.oomph.task.cli 1.0.0.201703021901
Bundle com.github.maybeec.oomph.task.fsrename 1.0.0.201612071753
Bundle com.github.maybeec.oomph.task.unzip 1.0.0.201612071748
Bundle org.eclipse.oomph.setup 1.9.0.v20180529-0916, build=3535, branch=df6403a4e53adefac88bf2032461e6674635a4cc
Bundle org.eclipse.oomph.setup.core 1.9.0.v20180529-0916, build=3535, branch=df6403a4e53adefac88bf2032461e6674635a4cc
Bundle org.eclipse.oomph.setup.jdt 1.9.0.v20180529-0916, build=3535, branch=df6403a4e53adefac88bf2032461e6674635a4cc
Bundle org.eclipse.oomph.setup.maven 1.9.0.v20180529-0916, build=3535, branch=df6403a4e53adefac88bf2032461e6674635a4cc
Bundle org.eclipse.oomph.setup.p2 1.9.0.v20180529-0916, build=3535, branch=df6403a4e53adefac88bf2032461e6674635a4cc
Bundle org.eclipse.oomph.setup.projects 1.9.0.v20180529-0916, build=3535, branch=df6403a4e53adefac88bf2032461e6674635a4cc
Bundle org.eclipse.oomph.setup.targlets 1.9.0.v20180529-0916, build=3535, branch=df6403a4e53adefac88bf2032461e6674635a4cc
Bundle org.eclipse.oomph.setup.workingsets 1.9.0.v20180529-0916, build=3535, branch=df6403a4e53adefac88bf2032461e6674635a4cc
Performing unzip unzip.cmder
Starting Unzip process
Archive: C:\CobiGenIDE\temp\cmder.zip
Destination: C:\CobiGenIDE\software
com.github.maybeec.oomph.task.unzip.core.exceptions.UnzipTaskException: java.io.FileNotFoundException: C:\CobiGenIDE\temp\cmder.zip (The system cannot find the file specified)
  at com.github.maybeec.oomph.task.unzip.core.impl.UnzipUtilImpl.unzip(UnzipUtilImpl.java:87)
  at com.github.maybeec.oomph.task.unzip.impl.unzipTaskImpl.perform(unzipTaskImpl.java:310)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.doPerformNeededSetupTasks(SetupTaskPerformer.java:3332)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.access$1(SetupTaskPerformer.java:3275)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil$1.run(SetupTaskPerformer.java:4502)
  at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2240)
  at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2267)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil.performNeededSetupTasks(SetupTaskPerformer.java:4496)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil.access$0(SetupTaskPerformer.java:4494)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.performNeededSetupTasks(SetupTaskPerformer.java:3266)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.performTriggeredSetupTasks(SetupTaskPerformer.java:3241)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.perform(SetupTaskPerformer.java:3141)
  at org.eclipse.oomph.setup.ui.wizards.ProgressPage$9.run(ProgressPage.java:575)
  at org.eclipse.oomph.setup.ui.wizards.ProgressPage$11$1.run(ProgressPage.java:701)
  at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.FileNotFoundException: C:\CobiGenIDE\temp\cmder.zip (The system cannot find the file specified)
  at java.io.FileInputStream.open0(Native Method)
  at java.io.FileInputStream.open(FileInputStream.java:195)
  at java.io.FileInputStream.<init>(FileInputStream.java:138)
  at com.github.maybeec.oomph.task.unzip.core.impl.UnzipUtilImpl.unzip(UnzipUtilImpl.java:38)
  at com.github.maybeec.oomph.task.unzip.impl.unzipTaskImpl.perform(unzipTaskImpl.java:310)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.doPerformNeededSetupTasks(SetupTaskPerformer.java:3332)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.access$1(SetupTaskPerformer.java:3275)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil$1.run(SetupTaskPerformer.java:4502)
  at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2240)
  at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2267)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil.performNeededSetupTasks(SetupTaskPerformer.java:4496)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil.access$0(SetupTaskPerformer.java:4494)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.performNeededSetupTasks(SetupTaskPerformer.java:3266)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.performTriggeredSetupTasks(SetupTaskPerformer.java:3241)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.perform(SetupTaskPerformer.java:3141)
  at org.eclipse.oomph.setup.ui.wizards.ProgressPage$9.run(ProgressPage.java:575)
  at org.eclipse.oomph.setup.ui.wizards.ProgressPage$11$1.run(ProgressPage.java:701)
  ... 1 more

Took 0 seconds.
There are failed tasks.
Press Back to choose different settings or Cancel to abort.
```

--- a/oomph/products/CobiGenDevelopmentIDE.setup
+++ b/oomph/products/CobiGenDevelopmentIDE.setup
@@ -467,7 +467,7 @@
   <setupTask
       xsi:type="com.github.maybeec.oomph.task:unzipTask"
       id="unzip.cmder"
-      excludedTriggers="STARTUP"
+      excludedTriggers="STARTUP MANUAL"
       predecessor="rc.cmder"
       filter=""
       zipFile="${tmp.download.location/cmder.zip}"

